week2/test_mlflow.py

```
@@ -0,0 +1,8 @@
+# Databricks notebook source
+import json
```

Ruff (0.7.0)

`json` imported but unused

Remove unused import: `json`

(F401)